src/OmniSharp.Roslyn.CSharp/Services/Refactoring/V2/CodeActionHelper.cs

```diff
@@ -47,7 +47,18 @@ public static class CodeActionHelper
             var diagnostics = semanticModel.GetDiagnostics();
             var location = GetTextSpan(request, sourceText);
```

Comment:
Probably better to call this "span" since there's a Roslyn type called "Location" which is being accessed below.